### PR TITLE
Added enums for ripple effect #328 

### PR DESCRIPTION
**Pull request by CarHel**
Added this enum on RippleAssist

`    public enum RippleEffect
    {
        Standard,
        Centered,
        None
    }`
and made according chages for it to work

### CODE REVIEW

**Comment by ButchersBoy** [comment]
OK, big problem here is this is a breaking API change.  Now, we are following semver, and will soon be moving to ver 2 as we have some breaking changes.  See https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/wiki/Version-2.0.0---Breaking-Changes

However, breaking changes is something I want to avoid unless it is really warranted to push the library forward.

So what are the other options for RippleAssist?

**Comment by CarHel** [comment]
Alright, didn't thought about that. I don't know how to revert changes, so i just copied back older versions and made the new changes.

File: MaterialDesignThemes.Wpf/RippleAssist.cs [hunk 1]
```diff
@@ -54,6 +54,35 @@ public static bool GetIsCentered(DependencyObject element)
...
+        /// </summary>
+        public static readonly DependencyProperty IsRippleDisabledProperty = DependencyProperty.RegisterAttached(
+            "IsRippleDisabled", typeof(bool), typeof(RippleAssist), new FrameworkPropertyMetadata(false, FrameworkPropertyMetadataOptions.Inherits));
+
+        /// <summary>
+        /// Set to <c>True</c> to disable ripple effect
+        /// </summary>
+        /// <param name="element"></param>
+        /// <param name="value"></param>
+        public static void SetIsRippleDisabled(DependencyObject element, bool value)
```
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/RippleAssist.cs]: Can we drop the "Ripple";  we don't need the noun as we are already in the RippleAssist class, and it  keeps it inline with IsCentered.  So, "IsDisabled".

**Comment by CarHel** [comment]
I made the changes, I'll try to be more careful next time. I don't understand what are the branch conflicts :/

**Comment by ButchersBoy** [comment]
re. merge conflicts, I did some changes on demo Buttons.xaml, so without me pulling your latest change I'm guessing around there.  

So, you need to figure out how to update your fork from master, fix the conflicts, and push the fixes back to GitHub.  Then we should be pretty much ready to pull into master.

**Comment by CarHel** [comment]
Guess i found out :D.

**Comment by ButchersBoy** [comment]
...your didn't need to close the PR...unless you find it easier starting clean

**Pull request closed without merging**